editor/src/components/canvas/canvas-strategies/relative-move-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,401 @@
+import { Modifiers, cmdModifier, shiftModifier } from '../../../utils/modifiers'
+import { windowPoint } from '../../../core/shared/math-utils'
+import { CanvasControlsContainerID } from '../controls/new-canvas-controls'
+import { mouseClickAtPoint, mouseDragFromPointWithDelta } from '../event-helpers.test-utils'
+import {
+  EditorRenderResult,
+  getPrintedUiJsCode,
+  makeTestProjectCodeWithSnippet,
+  renderTestEditorWithCode,
+} from '../ui-jsx.test-utils'
+
+const setupAndDrag = async (
+  project: string,
+  elementId: string,
+  x: number,
+  y: number,
+  modifiers?: Modifiers,
+): Promise<EditorRenderResult> => {
+  const renderResult = await renderTestEditorWithCode(project, 'await-first-dom-report')
+
+  const targetElement = renderResult.renderedDOM.getByTestId(elementId)
+  const targetElementBounds = targetElement.getBoundingClientRect()
+  const canvasControlsLayer = renderResult.renderedDOM.getByTestId(CanvasControlsContainerID)
+
+  const startPoint = windowPoint({ x: targetElementBounds.x + 5, y: targetElementBounds.y + 5 })
+  const dragDelta = windowPoint({ x, y })
```

Comment:
Heads up here - the use of `windowPoint` for these tests is a remnant of some older code that happened to require those types because of the utility functions we were using for them. The newer helper functions in `event-helpers.test-utils.ts` don't require `windowPoint`